### PR TITLE
DOC-2283: replace RedisBloom with Probabilistic

### PR DESCRIPTION
**Pull request by dwdougherty**
This commit is for the module de-branding effort.

### CODE REVIEW

File: docs/docs/Quick_Start.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ description: >
     "Quick start guide"
 ---
 
-# Quick Start Guide for RedisBloom
+# Quick Start Guide for RedisProbabilistic
```
Comment (elena-kolevska) [inline docs/docs/Quick_Start.md:9]:
```suggestion
# Quick Start Guide for Redis Probabilistic
```

File: docs/docs/Quick_Start.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ description: >
...
 
 ## Redis Cloud
 
-RedisBloom is available on all Redis Cloud managed services.  Redis Cloud Essentials offers a completely free managed database up to 30MB.
+Probabilistic is available on all Redis Cloud managed services.  Redis Cloud Essentials offers a completely free managed database up to 30MB.
```
Comment (elena-kolevska) [inline docs/docs/Quick_Start.md:13]:
```suggestion
Probabilistic is available on all Redis Cloud managed services.  Redis Cloud Fixed offers a completely free managed database of up to 30MB.
```

File: docs/docs/Clients.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: "Client Libraries"
 linkTitle: "Clients"
```
Comment (K-Jo) [inline docs/docs/Clients.md:3]: We should remove the clients.

File: docs/docs/Clients.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: "Client Libraries"
...
 weight: 4
 description: >
-    RedisBloom has several client libraries, written by the module authors and community members - abstracting the API in different programming languages.
+    Redis Probabilistic has several client libraries, written by the module authors and community members - abstracting the API in different programming languages.
```
Comment (K-Jo) [inline docs/docs/Clients.md:6]: Redis Probabilistic not a thing.  It's Redis Stack has several...

File: docs/docs/Configuration.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ title: "Configuration Parameters"
 linkTitle: "Configuration"
 weight: 5
 description: >
-    RedisBloom supports multiple module configuration parameters. All of these parameters can only be set at load-time.
```
Comment (K-Jo) [inline docs/docs/Configuration.md:6]: When we talk about the HOW to configure, we still need to use the module name. RedisBloom

File: docs/docs/Configuration.md [hunk 6]
```diff
@@ -28,7 +28,7 @@ From the command line:
 $ redis-server --loadmodule ./redisbloom.so [OPT VAL]...
 ```
 
-## RedisBloom configuration parameters
```
Comment (K-Jo) [inline docs/docs/Configuration.md:31]: Keep RedisBloom

File: docs/docs/Quick_Start.md [hunk 7]
```diff
@@ -6,15 +6,15 @@ description: >
     "Quick start guide"
 ---
 
-# Quick Start Guide for RedisBloom
+# Quick Start Guide for Redis Probabilistic
 
 ## Redis Cloud
 
-RedisBloom is available on all Redis Cloud managed services.  Redis Cloud Essentials offers a completely free managed database up to 30MB.
+Probabilistic is available on all Redis Cloud managed services.  Redis Cloud Fixed offers a completely free managed database up to 30MB.
```
Comment (K-Jo) [inline docs/docs/Quick_Start.md:13]: Redis Stack's probabilistic data structures are available on all Redis Cloud managed services

File: docs/docs/Quick_Start.md [hunk 8]
```diff
@@ -6,15 +6,15 @@ description: >
...
 
 [Get started here](https://redis.com/try-free/)
 
-## Launch RedisBloom with Docker
+## Launch Probabilistic with Docker
```
Comment (K-Jo) [inline docs/docs/Quick_Start.md:17]: Launch Redis Stack with Docker

File: docs/docs/_index.md [hunk 9]
```diff
@@ -8,7 +8,7 @@ type: docs
 [![Discord](https://img.shields.io/discord/697882427875393627?style=flat-square)](https://discord.gg/wXhwjCQ)
 [![GitHub](https://img.shields.io/static/v1?label=&message=repository&color=5961FF&logo=github)](https://github.com/RedisBloom/RedisBloom/)
 
-RedisBloom contains a set of useful probabilistic data structures. Probabilistic data structures allow developers to control the accuracy of returned results while gaining performance and reducing memory.
```
Comment (K-Jo) [inline docs/docs/_index.md:11]: Redis Stack contains a set of

**Review by K-Jo — CHANGES_REQUESTED** [review]

File: docs/docs/Clients.md (hunk 3 above)
Comment (dwdougherty) [reply docs/docs/Clients.md:3]: Hi Pieter. Apologies, but I'm not clear what you're asking me to do with this comment. Would you please clarify for me?

File: docs/docs/Quick_Start.md (hunk 8 above)
Comment (dwdougherty) [reply docs/docs/Quick_Start.md:17]: Change made. I'll update the PR once all requested changes have been made.

File: docs/docs/Clients.md (hunk 3 above)
Comment (K-Jo) [reply docs/docs/Clients.md:3]: We should not have client library specific pages for modules on redis.io. We can keep this part of the readme of the module repository

Comment (elena-kolevska) [reply docs/docs/Clients.md:3]: @dwdougherty don't worry about this one, it's a part of my fork

Comment (dwdougherty) [reply docs/docs/Clients.md:3]: Thanks Elena. Resolving.

**Comment by dwdougherty** [comment]
Changes were covered by other commits.

**Pull request closed without merging**